app/presenters/hyrax/manifest_components.rb

```diff
@@ -0,0 +1,78 @@
+module Hyrax
+  module ManifestComponents
+    delegate :rendering_ids, to: :solr_document
+
+    def manifest_url
+      manifest_helper.polymorphic_url([:manifest, self])
+    end
+
+    # IIIF rendering linking property for inclusion in the manifest
+    #
+    # @return [Array] array of rendering hashes
+    def sequence_rendering
```

Comment:
Nothing appears to be using this method. Is that right?

Reply:
The `iiif_manifest` gem calls it on the object it's given (in this case the presenter).

Reply:
It's used by `iiif_manifest` to build the manifest, ditto the other one.